modules/storages/db/migrate/20240405135016_update_access_management_of_configured_one_drive_storages.rb

```diff
@@ -0,0 +1,11 @@
+class UpdateAccessManagementOfConfiguredOneDriveStorages < ActiveRecord::Migration[7.1]
+  def up
+    execute <<-SQL.squish
+      UPDATE storages
+      SET provider_fields = jsonb_set(provider_fields, '{automatically_managed}', 'false', true)
+      WHERE provider_type = 'Storages::OneDriveStorage' AND provider_fields->>'automatically_managed' is null;
```

Comment:
Just being curious: why not do it for both NextcloudStorage and OneDriveStorage?

Reply:
Got it. Thanks for clarification!